packages/reakit/src/Toolbar/__examples__/ToolbarWithMenu/__tests__/index-test.tsx

```diff
@@ -0,0 +1,78 @@
+import * as React from "react";
+import { render, press, click } from "reakit-test-utils";
+import ToolbarWithMenu from "..";
+
+describe("<ToolbarWithMenu />", () => {
+  test("renders toolbar items with closed menu", () => {
+    const { getByText: text } = render(<ToolbarWithMenu />);
+
+    expect(text("Apples")).toBeVisible();
+    expect(text("Oranges")).toBeVisible();
+    expect(text("Other Fruits")).toBeVisible();
+
+    expect(text("Pears")).not.toBeVisible();
+    expect(text("Kiwis")).not.toBeVisible();
+    expect(text("Lemons")).not.toBeVisible();
+  });
+
+  it("can navigate toolbar items through keyboard", () => {
```

Comment:
I tend to use always `test`. I think we can enforce this with [this eslint plugin](https://github.com/jest-community/eslint-plugin-jest/blob/master/docs/rules/consistent-test-it.md), but this can be done in another PR. :)
```suggestion
  test("can navigate toolbar items through keyboard", () => {
```